server/routes/auth/auth.js

```diff
@@ -8,22 +8,30 @@ router.get('/login', passport.authenticate('OAuth2', {scope: 'read:user'}, (req,
     res.redirect(redirectURI);
 }));
 
-const checkIsAdmin = async (id) => {
-  const { rows } = await db.query(`
-  SELECT COUNT(user_id)
-  FROM jobs.department_head as ramads
-  WHERE ramads.user_id = $1
-  `, [id]);
+// const checkIsAdmin = async (id) => {
```

Comment:
why did you comment it?

Reply:
we dont use that anymore, look at the function that calls after the Aut finished